### DISCUSSION TITLE
Where are the files for building the library?

### DISCUSSION THREAD

**Discussion by TheEpicFace007** [ImageMagick/ImageMagick#3119] [Help] [answered]
I keep searching for the file that the docs(https://imagemagick.org/script/advanced-windows-installation.php) are saying where to look for but I don't find it. Where do I need to go?

**Comment by urban-warrior** [comment]
See https://imagemagick.org/script/install-source.php#windows.

**Comment by TheEpicFace007** [comment]
I don't find the `configure` directory that I am supposed to find the build solution.

**Comment by TheEpicFace007** [comment]
I use window

**Comment by TheEpicFace007** [comment, marked as answer]
edit: I didn't realised that I had to open the **VisualMagick folder**. I tough I had to open the ImageMagick folder. I read too quickly  the doc
